applications/tasks/src/kz_tasks_scheduler.erl

```diff
@@ -143,25 +147,31 @@ worker_finished(TaskId=?NE_BINARY, TotalSucceeded, TotalFailed, Output=?NE_BINAR
   when is_integer(TotalSucceeded), is_integer(TotalFailed) ->
     _ = gen_server:call(?SERVER, {'worker_finished', TaskId, TotalSucceeded, TotalFailed}),
     {'ok', CSVOut} = file:read_file(Output),
-    AName = ?KZ_TASKS_ANAME_OUT,
-    attempt_upload(TaskId, AName, CSVOut, Output, 3).
+    Max = ?UPLOAD_ATTEMPTS,
+    attempt_upload(TaskId, ?KZ_TASKS_ANAME_OUT, CSVOut, Output, Max, Max).
 
-attempt_upload(_TaskId, _AName, _, _, 0) ->
-    lager:error("failed saving ~s/~s: 3rd conflict", [_TaskId, _AName]),
+attempt_upload(_TaskId, _AName, _, _, 0, _) ->
```

Comment:
Why not start Retries at 1 and go until `Max, Max)` here? Then you don't have to do math at line 159 below